### ISSUE TITLE
Button does not show label after being nulled before

### ISSUE THREAD

**Issue by er453r** [ianharrigan/haxeui#319]
The problem lies here: https://github.com/ianharrigan/haxeui/blob/master/haxe/ui/toolkit/controls/Button.hx#L360

The solution is to add `_label.visible = true;` before `organiseChildren();`

**Comment by ianharrigan** [comment]
That seems strange, if you null (destroy) the label, then it should be recreated and when components are recreated they should start off as visible. Why are you nulling the label btw?

**Comment by ianharrigan** [comment]
Ah, sorry, i understand... You mean when you set the .text to null and then back to a value... gotcha... :)

**Comment by er453r** [comment]
Yes, I've meant `nulling` the text field, sorry for the confusion

**Comment by ianharrigan** [comment]
Should be fixed now in latest haxelib version.
